src/libcollectionstest/enum_set.rs

```diff
@@ -220,6 +222,7 @@ fn test_overflow() {
     #[allow(dead_code)]
     #[derive(Copy, Clone)]
     #[repr(usize)]
+    #[rustfmt_skip]
```

Comment:
Seems like this caused the build problem.
The error can be seen [here](https://travis-ci.org/rust-lang/rust/builds/132171217#L3552).